### PR TITLE
French language support

### PR DESCRIPTION
**Pull request by HumorousFool**
### Describe in detail what your pull request accomplishes
This PR cleans up the existing language files and adds a new one for french.

### Checklist
- [x] Proper internationalization
- [x] Tested

### CODE REVIEW

**Comment by TylerS1066** [comment]
I don't speak french, but it looks good to me!  I do have to review the removed and changed internationalization strings.

File: modules/Movecraft/src/main/resources/localisation/movecraftlang_cz.properties [hunk 1]
```diff
@@ -35,56 +35,31 @@ Detection\ -\ Craft\ too\ small=Detekce selhala\! Stroj byl prilis maly. Minimum
 Detection\ -\ Craft\ too\ large=Detekce selhala\! Stroj byl prilis velky. Maximum pro tento typ je %d
 Detection\ -\ Forbidden\ block\ found=Detekce selhala\! Ve stroji byl nalezen zakazany blok.
 Detection\ -\ Failed\ -\ Already\ commanding\ a\ craft=Uz jeden stroj ridis\!
-Detection\ -\ Failed\ Craft\ is\ already\ being\ controlled=Stroj uz je nekym ovladan\!
```
Comment (TylerS1066) [inline modules/Movecraft/src/main/resources/localisation/movecraftlang_cz.properties:38]: This is something that should be used, but isn't currently.

File: modules/Movecraft/src/main/resources/localisation/movecraftlang_cz.properties [hunk 2]
```diff
@@ -35,56 +35,31 @@ Detection\ -\ Craft\ too\ small=Detekce selhala\! Stroj byl prilis maly. Minimum
...
 Translation\ -\ Failed\ Craft\ hit\ minimum\ height\ limit=Stroj dosahl minimalniho limitu vysky
-Translation\ -\ Failed\ Craft\ over\ water=Tento stroj se nemuze pohybovat nad vodou\!
-Translation\ -\ Craft\ collision=Kolize stroje
-Internal\ -\ Error\ -\ Proccessor\ thread\ encountered\ an\ error=Async uloha vygenerovala vyjimku.
+Internal\ -\ Error\ -\ Processor\ thread\ encountered\ an\ error=Async uloha vygenerovala vyjimku.
 Startup\ -\ Number\ of\ craft\ files\ loaded=Nacteno %d souboru stroju
-Startup\ -\ Error\ parsing\ CraftType\ file=Chyba pri parsovani souboru TypStroje z adresare: %s
 Startup\ -\ Error\ -\ Reload\ error=Movecraft je nekompatibilni s prikazem reload. Movecraft se vypnul a bude restartovan az s restartem serveru.
 Startup\ -\ Error\ -\ Disable\ warning\ for\ reload=Pokud chcete pouzit prikaz reload a Movecraft, muzete tuto kontrolu vypnout v config.yml nastavenim 'safeReload: false'
 Startup\ -\ Enabled\ message=[V %S] byl nacten.
```
Comment (TylerS1066) [inline modules/Movecraft/src/main/resources/localisation/movecraftlang_cz.properties:48]: Not needed for this PR, but I would like to remove internationalization from startup and console messages.  Unfortunately, we are unable to provide translations for all of our wiki material and as such we expect server owners to have a passable understanding of English console messages.

File: modules/Movecraft/src/main/resources/localisation/movecraftlang_cz.properties [hunk 3]
```diff
@@ -35,56 +35,31 @@ Detection\ -\ Craft\ too\ small=Detekce selhala\! Stroj byl prilis maly. Minimum
...
-Player-\ Craft\ is\ sinking=Vas stroj je prilis poskozen a potapi se\! Opustit palubu\!
-Player-\ Error\ -\ You\ do\ not\ have\ a\ craft\ to\ release!=Nemas zadny stroj, ktery by jsi mohl opustit\!
-Player\ -\ Error\ -\ Already\ piloting\ craft=Uz jeden stroj ridis\!
-MCStats\ -\ \:(\ -\ Admin\ has\ opted\ out=Vybrali jste odhlaseni od odesilani dat na mcstats.org. Vyvojar Movecraftu by byl velmi vdecny za vase data.
-MCStats\ -\ Thank\ you\ message=Statistiky byly uspesne odeslany na mcstats.org. Dekujeme za vasi podporu.
-MCStats\ -\ Error\ -\ Unable\ to\ upload\ stats=Nemohu odeslat statistiky na mcstats.org. Prosim, zkontrolujte sve pripojeni k siti.
+Player\ -\ Craft\ is\ sinking=Vas stroj je prilis poskozen a potapi se\! OPUSTIT PALUBU\!
+Player\ -\ Error\ -\ You\ do\ not\ have\ a\ craft\ to\ release!=Nemas zadny stroj, ktery by jsi mohl opustit\!
 Release\ -\ Player\ has\ left\ craft=Opustil jsi svuj stroj\! Mas 15 vterin na navrat do stroje, jinak automaticky pustis ovladani.
-Release\ -\ Craft\ has\ been\ released\ message=Pustil jsi ovladani sveho stroje.
```
Comment (TylerS1066) [inline modules/Movecraft/src/main/resources/localisation/movecraftlang_cz.properties:63]: This message is still used.

**Review by TylerS1066 — CHANGES_REQUESTED** [review]
I've not finished looking through the changes, but I've already found a number of issues.  Please review all removals you've made for usage in Movecraft.

**Comment by HumorousFool** [comment]
I've re-added those messages double checked the rest of my deletions.

**Review by TylerS1066 — APPROVED** [review]

**Pull request merged**